src/gui/MainWindow.cpp

```diff
@@ -911,7 +926,6 @@ void MainWindow::switchToDatabases()
 void MainWindow::switchToSettings(bool enabled)
 {
     if (enabled) {
-        m_ui->settingsWidget->loadSettings();
```

Comment:
@phoerious This line removal breaks the Browser Integration settings.

Reply:
Alright, good find. I remember I had some weird issues with settings not being loaded which I fixed, though. I will go through the diff and see if there are any incidental changes that shouldn't be there.

Reply:
Fixed.